docs/documentation/cn/sql-reference/sql-statements/Data Definition/ALTER TABLE.md

```diff
@@ -232,7 +232,12 @@
         PROPERTIES ("bloom_filter_columns"="k1,k2,k3");
 
     12. 修改表的Colocate 属性
-        ALTER TABLE example_db.my_table set ("colocate_with"="t1");
+
+        ALTER TABLE example_db.my_table set ("colocate_with" = "t1");
+
+    13. 将表的分桶方式由 Random Distribution 改为 Hash Distribution
```

Comment:
Do we really need user to send a command to convert distribution? I think we can do it when we upgrade.

Reply:
I thought about convert it when saving meta image or loading meta image.
But it is very easy to forget to modify someplace, especially when replaying some edit logs related to distribution.

So I think using a command and handle it like a alter table job is more safety.

Reply:
ok